.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"trailingComma": "es5",
+	"singleQuote": true,
+	"semi": false,
+	"printWidth": 100,
+	"tabWidth": 2,
+	"useTabs": true
+}
```

Comment:
the organise imports plugin didnt seem to work here? would be good to get it working